docs/architecture/basic-normalization.md

```diff
@@ -63,6 +63,13 @@ Airbyte uses the types described in the catalog to determine the correct type fo
 
 ### Nesting
 
+{% hint style="warning" %}
+Normalization is still a work in progress and is very basic for the moment. We are still working on improving it.
+Nested Objects and Arrays are not currently handled by Normalization (only flat jsons are properly mapped out as a table).
+You can follow progress as part of this
+[issue](https://github.com/airbytehq/airbyte/issues/886) and its related children.
+{% endhint %}
+
 Basic Normalization attempts to expand any nested arrays or objects it receives into separate tables in order to allow more ergonomic querying of your data.
```

Comment:
should this be removed?

Reply:
That's what normalization should do once we fix FE/BE issues with the catalog so we can keep it for the meantime?

Reply:
it's just that it conflicts with the text you just added about arrays not yet being supported, which could confuse a user. I'd say we should remove it or edit it then re-add it in the PR to support nested